neuron/src/lib/Neuron/Zettelkasten/Graph.hs

```diff
@@ -100,6 +102,9 @@ getZettels = G.getVertices
 getZettel :: ZettelID -> ZettelGraph -> Maybe Zettel
 getZettel = G.findVertex
 
+getConnections :: ZettelGraph -> Map.Map ZettelID (Map.Map ZettelID (Maybe Connection))
+getConnections = LAM.adjacencyMap . G.getGraph
```

Comment:
Move under the `where` clause of where it is used.

Reply:
Actually, just move this to Algorithm.hs (where it will be polymorphic) and use directly as `G.getGraphAdjacencyMap`.

Reply:
Actually, nevermind. There are larger problems with this module. I'll refactor later.